test/parallel/test-require-package-json-directory.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+
+try {
+  require(fixtures.path('module-package-json-directory'));
+  process.exit(1);
```

Comment:
Would it be better to `assert.fail` here instead? I'm thinking from the point of view of how it might look in the CI if the test failed here.

Reply:
Can't we just use `assert.throws()`?

Reply:
it would just be caught by the `catch` below.